metadata.yaml

```diff
@@ -41,12 +41,22 @@ provides:
     interface: mysql_client
   osm-mysql:
     interface: mysql
+  metrics-endpoint:
+    interface: prometheus_scrape
+  grafana-dashboard:
+    interface: grafana_dashboard
```

Comment:
just confirming that we dont want to limit these relations as well?

Reply:
I'm not sure yet - I'll leave for another PR